### ISSUE TITLE
Error for "Create Shipment" Method

### ISSUE THREAD

**Issue by Giorgio-Montagnoli** [EasyPost/easypost-csharp#67]
Hi,

at this url
https://www.easypost.com/docs/api/csharp.html#shipments-create-codesample
I find a C# sample that I have implemented in this way
```
static void Main(string[] args)
{
  ClientManager.SetCurrent(() => new Client(new ClientConfiguration("xjv...VTQ")));
 
  var fromAddress = new Address
  {
    name = "EasyPost",
    street1 = "417 Montgomery Street",
    street2 = "5th Floor",
    city = "San Francisco",
    state = "CA",
    zip = "94104",
    country = "US",
    phone = "4153334444",
    email = "support@easypost.com"
  };
 
  // verifications: non genera errore
  // strictVerifications: solleva una EasyPost.HttpException
  fromAddress.Create(strictVerifications: new List<string>() { "delivery" });
 
  var toAddress = new Address
  {
    name = "Dr. Steve Brule",
    street1 = "179 N Harbor Dr",
    city = "Redondo Beach",
    state = "CA",
    zip = "90277",
    country = "US",
    phone = "4153334444"
  };
 
  toAddress.Create(strictVerifications: new List<string>() { "delivery" });
 
  var parcel = new Parcel
  {
    length = 20.2,
    width = 10.9,
    height = 5.0,
    weight = 140.8
  };
 
  Console.WriteLine(parcel.predefined_package);
 
  var shipment = new Shipment
  {
    to_address = toAddress,
    from_address = fromAddress,
    parcel = parcel,
    customs_info = new CustomsInfo 
    { 
      contents_type = "merchandise",
      customs_certify = "false",
      non_delivery_option = "return",
      restriction_type = "none",
      customs_items = new List<CustomsItem>
      {
        new CustomsItem
        {
          description = "Many, many EasyPost stickers.",
          hs_tariff_number = "123456",
          origin_country = "US",
          quantity = 1,
          value = 879,
          weight = 140,
        }
      }
    }
  };
 
  shipment.Create();
 
  Console.ReadLine();
}
```
But I've got this error
```
'DateTimeInvalidLocalFormat'
Additional information: This can happen when calling DateTime.ToString using the 'z' format specifier, which will include a local time zone offset in the output. In that case, either use the 'Z' format specifier, which designates a UTC time, or use the 'o' format string, which is the recommended way to persist a DateTime in text. This can also occur when passing a DateTime to be serialized by XmlConvert or DataSet. If using XmlConvert.ToString, pass in XmlDateTimeSerializationMode.RoundtripKind to serialize correctly. If using DataSet, set the DateTimeMode on the DataColumn object to DataSetDateTime.Utc.
```
You should be able to fix it, here
https://github.com/EasyPost/easypost-csharp/blob/master/EasyPost/Request.cs#L106
if I read the error correctly.

Can you please provide us a feedback?

**Comment by Giorgio-Montagnoli** [comment]
I made a pull request to try to fix this error here #68

**Comment by Giorgio-Montagnoli** [comment]
I made another pull for another error

Fix the "Validation failed: Weight is not a number" error
"When you are not in US (we are in Italy) you got this error if you use a number with decimal values.
I could fix setting the Thread Culture in my program, but should be better to force it in your code, cause the data you sent is always in "en-US" format."
